migrations/versions/e47fde01cc62_add_csm_sequence.py

```diff
@@ -0,0 +1,23 @@
+"""add csm sequence
+
+Revision ID: e47fde01cc62
+Revises: e505f2e15499
+Create Date: 2022-10-06 16:24:05.695978
+
+"""
+from alembic import op
+
+
+# revision identifiers, used by Alembic.
+revision = "e47fde01cc62"
+down_revision = "e505f2e15499"
```

Comment:
Something has gone wrong here with the migrations.  This migration and the one for adding the index to the barcodes_group table both say they revise the same source migration and that their down_revision is the same.  This will break the migrations process.